src/signals/incident-management/containers/Dashboard/components/Filter/Filter.tsx

```diff
@@ -0,0 +1,118 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
+import { useCallback, useEffect, useMemo } from 'react'
+
+import { useForm } from 'react-hook-form'
+import { useSelector } from 'react-redux'
+
+import { makeSelectDepartments } from '../../../../../../models/departments/selectors'
+import { generateParams } from '../../../../../../shared/services/api/api'
+import type { Department } from '../../../../../../types/api/incident'
+import {
+  priorityList,
+  punctualityList,
+  stadsdeelList,
+} from '../../../../definitions'
+
+/**
+ * make fetchGraphData required when fetching for diagrams
+ */
+type Props = {
```

Comment:
Misschien dat ik hem even kwijt ben, maar deze props wordt niet meegegeven aan filter in dashboard. En filter wordt nergens anders aangemaakt. Waarom deze prop dan?

Reply:
omdat dit de callback functie wordt die de data gaat aanroepen, wanneer die gebouwd gaat worden